src/main/java/com/couchbase/lite/support/security/SymmetricKey.java

```diff
@@ -216,11 +273,7 @@ private Cipher getCipher(int mode, byte[] iv) throws SymmetricKeyException {
      * @throws SymmetricKeyException
      */
     public Encryptor createEncryptor() throws SymmetricKeyException {
```

Comment:
I doesn't do much thing exception new Encryptor. iOS has this method which does some stuff so just tried to stay as close as the iOS but no problems, we can remove.